package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "@doordash/eslint-config-doordash",
-  "version": "1.0.11",
+  "version": "2.0.0",
   "description": "DoorDash's ESLint config, following our styleguide",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "format": "npm run lint -- --fix",
+    "lint": "eslint packages/"
```

Comment:
This is project specific and will not work for all projects (since all projects won't be lerna repos).  Probably something best left up to the package to define.

Reply:
ah yeah... will kill this, thanks